app/assets/javascripts/host_edit.js

```diff
@@ -584,6 +584,8 @@ $(document).on('change', '.interface_domain', function() {
       .find('.interface_ip6')
   );
   interface_domain_selected(this);
+  clearIpField('.interface_domain', '.interface_ip');
+  clearIpField('.interface_domain', '.interface_ip6');
```

Comment:
since `clearError` is being called also in `clearIpField` we can remove the code above:
```js
$(document).on('change', '.interface_domain', function() {
  interface_domain_selected(this);
  clearIpField(this, '.interface_ip');
  clearIpField(this, '.interface_ip6');
  reload_host_params();
});
```